apps/datahub/src/app/search/search-results/search-results.component.ts

```diff
@@ -8,9 +8,20 @@ import { SearchFacade } from 'geonetwork-ui'
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SearchResultsComponent {
-  constructor(protected searchFacade: SearchFacade) {}
+  pageSize = 18
+
+  constructor(
+    protected searchFacade: SearchFacade,
+    protected searchService: SearchService
+  ) {
+    this.searchFacade.setPageSize(this.pageSize)
+  }
 
   onShowMore() {
```

Comment:
I think this is not used anymore and can be removed.

Reply:
Yes, good catch :slightly_smiling_face: